index.rst

```diff
@@ -419,15 +463,20 @@ Clients
 Portal
 ~~~~~~
 
-When a user first logs into the portal, they will be redirected to the token issuer. They may select
-either NCSA as their Identity Provider or their home institution. CILogon executes the login,
-ultimately returning information about who the user is at NCSA to the portal aspect through
-CILogon's OpenID Connect interface and the token's ``sub`` claim. This provides the Portal aspect
-with an access token and a refresh token.
+When a user first logs into the portal, the `token proxy <#token-proxy>`__ will intercept the login
+and redirect them to CILogon. They may select either NCSA as their Identity Provider or an
+associated external identity. CILogon executes the login, ultimately returning information about
+who the user is at NCSA to the token proxy through CILogon's OpenID Connect interface, and an
+identity token with the proper `identity token claims <#identity-token-claims>`__. The token
+proxy, through the `token issuer <#token-issuer>`__ component, will then reissue the token with
+the same claims but with a 24-hour lifetime.
+
+The portal will have access to that token, and setup it's own user session based on the information
```

Comment:
Change to "The portal will be passed that token in an HTTP request header.  It can use the information in the token to customize its behavior, and it passes that token in requests to the API aspect as an OAuth 2.0 Bearer token via the HTTP `Authorization` header, according to the OAuth 2.0 Specification:"